.travis.yml

```diff
@@ -1,10 +1,11 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.5.1
-  - 2.4.4
+  - 2.6.5
+  - 2.5.7
+  - 2.4.9
+  - 2.3.8
   - 2.3.4
```

Comment:
Is there any reason to keep this version?

Reply:
Nope. Don't think so. Ruby 2.3.8 was the last 2.3.x release. I think that should suffice.

Reply:
Thank you for your review. I removed it.